components/typeform/actions/update-form/update-form.mjs

```diff
@@ -1,13 +1,14 @@
 import typeform from "../../typeform.app.mjs";
-import common from "../common.mjs";
+import utils from "../utils.mjs";
+
+const { reduceProperties } = utils;
 
 export default {
   key: "typeform-update-form",
   name: "Update a Form",
```

Comment:
should probably call this "Update Form Title"

Reply:
sorry, we should call the component itself "Update Form Title" (the .mjs file and parent folder"